simpleclient/src/main/java/io/prometheus/client/Summary.java

```diff
@@ -137,15 +168,15 @@ public double observeDuration() {
    * {@link SimpleCollector#remove} or {@link SimpleCollector#clear}.
    */
   public static class Child {
-    public static class Value {
-      public final double count;
-      public final double sum;
-      public final CKMSQuantiles quantiles;
+    private static class Value {
```

Comment:
It's for users to pull out information. It should be immutable.

A array/map snapshot would be good.